spring-boot-modules/spring-boot-properties/pom.xml

```diff
@@ -122,7 +123,7 @@
     </profiles>
 
     <properties>
```

Comment:
What's the purpose of introducing these changes, and how are they relevant to current task?

Reply:
I don't understand exactly what do you mean. I don't see any other changes than this:
![image](https://user-images.githubusercontent.com/12120641/231079694-62c8453d-680e-4438-90bb-5714b85703ea.png)

Reply:
That's ok

Reply:
On master we have version 2022.0.3.
Could you update this also here, if there's no other reason for 2022.0.1?

Reply:
The master version is 2021.0.3
![image](https://user-images.githubusercontent.com/12120641/231673090-3c42fe70-c076-4000-a070-1ee8ef196d37.png)
https://github.com/eugenp/tutorials/blob/master/spring-boot-modules/spring-boot-properties/pom.xml 
Do you want to add the version 2021.0.3 or keep the latest version which I added 2022.0.1?

Reply:
My mistake, I didn't see clearly it's **2021**.0.3, leave the new one.